input/assets/css/_cart.scss

```diff
@@ -96,6 +96,13 @@
 
 .edit-delete-section {
 	padding: 0.5em 0;
+
+    // Inline contextual rules
```

Comment:
Didn't have a better idea on where to put this, so I made it contextual to the selector for now